daml-lf/scenario-interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/ScenarioRunner.scala

```diff
@@ -192,14 +192,18 @@ final case class ScenarioRunner(
   private def lookupKey(
       gk: GlobalKey,
       committers: Set[Party],
-      cb: SKeyLookupResult => Boolean,
+      canContinue: SKeyLookupResult => Boolean,
```

Comment:
Wasn't `cb` clear enough? 😉
Did you figure out what it stands for?

Reply:
`callback` presumably but that isn’t much more useful than `cb` so I’ve renamed it to what it actually does :slightly_smiling_face:

Reply:
Yeah, `callback` sounds very reasonable. Thanks. I haven't done enough JS yet to make the word omnipresent in my head. And I agree that `callback` carries almost as little information as `cb`.